app/controllers/users/registrations_controller.rb

```diff
@@ -1,7 +1,6 @@
 module Users
   class RegistrationsController < Devise::RegistrationsController
     before_action :confirm_two_factor_authenticated, only: [:edit, :update, :destroy_confirm]
```

Comment:
The `destroy` action should not have been removed here. We can't allow a user to be deleted unless they're signed in. Note that the `destroy` action is defined in the Devise controller, so even though it's not in our app, it's still present because we're inheriting from the Devise controller.